### ISSUE TITLE
Allow manual entry of spatial bounds for Items searches

### ISSUE THREAD

**Issue by trevorlang** [radiantearth/stac-browser#176]
In addition to the map, a user should be able to manually enter spatial boundaries to filter items. It is a common use case to want to copy/paste spatial bounds from another source when searching for items.

**Comment by m-mohr** [comment]
How would you add that? Maybe two tabs to switch between map and coordinate form?

**Comment by trevorlang** [comment]
That seems like a nice way to handle it. I could see a set of tabs or a radio buttons that revealed either the form field or the map. 

The form field should probably have some sort of tooltip or label that indicated the required formatting. 

Validation would be great as well, if that's possible to achieve.

**Comment by chiarch84** [comment]
I think it would be also useful to be able to draw a specific area on the map and not only filtering by bounding box. And this could be done either by drawing on the map or by entering the polygon coordinates as text as suggested above.

**Comment by m-mohr** [comment]
@chiarch84 The main issue here is API support, by default only bbox is part of the spec. For geometries, you need the additional support of other conformance classes (e.g. intersects).

**Comment by chiarch84** [comment]
hmm, I thought the _intersects_ field was already part of the specs. In our APIs for example, that are based on Elastic-Fast-STACAPIs, I have the intersect field available from the API GUI and we didn't write the code for it, so I thought it was already meant to be like this.
![image](https://user-images.githubusercontent.com/98519154/214263143-d3fd1084-ce83-4779-a9b4-2017f73f999f.png)

**Comment by m-mohr** [comment]
Yeah, I updated my answer quickly afterwards. It is indeed part of the main spec, I thought it's just part of query or filter. 
So yeah, would be nice to have additional support. Is anyone aware of a good leaflet plugin that can be used to draw polygons?

**Comment by chiarch84** [comment]
What about https://leaflet.github.io/Leaflet.draw/docs/leaflet-draw-latest.html

**Comment by m-mohr** [comment]
Yes, that might be another alternative, indeed.

**Comment by ajgrowney** [comment]
Would there be any opportunities to reintroduce this as an option with the current state of the UI?

**Comment by m-mohr** [comment]
@ajgrowney Such a feature did never exist, but there's an open PR which you can try out: https://github.com/radiantearth/stac-browser/pull/389 Unfortunately, I did not get to reviewing it yet.

**Comment by ajgrowney** [comment]
Good to know! I'll take a look, thanks!
